evalml/pipelines/time_series_pipeline_base.py

```diff
@@ -183,12 +183,18 @@ def predict(self, X, objective=None, X_train=None, y_train=None):
             y_train (pd.Series or None): Training labels.
 
         Raises:
-            ValueError: If final component is not an Estimator.
+            ValueError: If X_train and/or y_train has no value or if final component is not an Estimator.
 
         Returns:
             Predictions.
         """
-        X_train, y_train = self._convert_to_woodwork(X_train, y_train)
+        try:
+            print("HELLO")
```

Comment:
need to remove this print

Reply:
I am glad to see that someone else does this as well